src/containers/tud/schemas/NightTimeActivitySchema.js

```diff
@@ -21,7 +15,7 @@ const {
   WAKE_UP_COUNT,
 } = PROPERTY_CONSTS;
 
-const createSchema = (pageNum :number) => ({
+const createSchema = (pageNum :number, trans :(string, ?Object) => Object) => ({
```

Comment:
I've seen you define `trans :(string, ?Object) => Object` in many places. Please create a type for it and reuse it.